src/main/java/org/apache/sysds/conf/DMLConfig.java

```diff
@@ -19,21 +19,6 @@
 
 package org.apache.sysds.conf;
 
-import java.io.ByteArrayInputStream;
```

Comment:
A minor comment: The styling related or formatting changes can overload the changeset and difficult to glance at which specific line has changed.

For example, https://github.com/apache/systemds/commit/bf4ba16b9aaa9afee20a3f1c03b0ff49c5346a9d